ui/ts/models/status.ts

```diff
@@ -150,6 +160,16 @@ module Models {
         return Object.keys(this._data.result()).sort();
       }
 
+      public GetAllStatuses(): Proto.NodeStatus[] {
```

Comment:
Indeed it could and should, but this entire file is going to be overhauled very shortly so it can be deferred.